tests/tools/execution/test_introspection.py

```
@@ -45,22 +48,22 @@ def test_describe_workflow_valid_workflow(self) -> None:
+        res = describe_process("core.arithmetic.add")
         assert res["group"] == "aiida.calculations"
         assert res["entry_point"] == "core.arithmetic.add"
         assert "x" in res["required_inputs"]
         assert "y" in res["required_inputs"]
 
-    def test_describe_workflow_missing_entry_point(self) -> None:
-        """Ensure `describe_workflow` raises ValueError when entry point is not found."""
+    def test_describe_process_missing_entry_point(self) -> None:
+        """Ensure `describe_process` raises ValueError when entry point is not found."""
         with pytest.raises(
             ValueError, match="not found in 'aiida.workflows' or 'aiida.calculations'"
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)